### PR TITLE
Add Astronomer recap post

### PR DESCRIPTION
**Pull request by bitsondatadev**

### CODE REVIEW

File: _posts/2022-12-20-trino-summit-2022-astronomer-recap.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+## Recap
+
+This talk covers a fictional trading scenario that initially follows a
+traditional warehousing approach to storing their data. This architecture has
+data sources that are queried and submitted as raw data into a centralized
+warehouse. Within the warehouse itself, the raw data in transformed into data
+ready to be consumed.
+
+This model enforces centralization which requires one team that is removed from
+the business use case to run the platform and build the integration between
```
Comment (colebow) [inline _posts/2022-12-20-trino-summit-2022-astronomer-recap.md:43]: I think the "one team that is removed from the business use case" is tough-ish to parse here. Maybe offset it with commas or reword.

File: _posts/2022-12-20-trino-summit-2022-astronomer-recap.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+producers and consumers. As source databases evolve, the central data team must
+keep up with these changes and are commonly a bottleneck.
+
+Trino allows you to move the queries as close as possible to the federated data
+sources. This removes the labor-intensive process of moving data into stages
+before ingesting it into a central warehouse gets abstracted away. This doesn't
```
Comment (colebow) [inline _posts/2022-12-20-trino-summit-2022-astronomer-recap.md:49]: One of "This removes" and "gets abstracted away" needs to go.

File: _posts/2022-12-20-trino-summit-2022-astronomer-recap.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+mean that data movement isn't no longer necessity, but it does mean that the
```
Comment (colebow) [inline _posts/2022-12-20-trino-summit-2022-astronomer-recap.md:50]:
```suggestion
mean that data movement is no longer a necessity, but it does mean that the
```

File: _posts/2022-12-20-trino-summit-2022-astronomer-recap.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+concern. 
+
+Without investing into more resources, your data professionals are able to work
+closely with producers and stakeholders with a shared understanding of the
+domain. This increases data literacy and data availability throughout your
+organization.
+
+Trino is not only for fast adhoc analytics with a human in the loop, but now 
+provides a fault-tolerant execution mode that enables it to run resource
+intensive batch jobs. This paired with the federation capabilities make Trino
```
Comment (colebow) [inline _posts/2022-12-20-trino-summit-2022-astronomer-recap.md:61]:
```suggestion
intensive batch jobs. This, paired with the federation capabilities, make Trino
```

File: _posts/2022-12-20-trino-summit-2022-astronomer-recap.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+fast adhoc analytics queries as opposed to long-running, resource intensive
+queries commonly classified as batch or ETL queries. This is due to the fact
+that Trino kills queries that run out of resources in favor of lower latency.
+Earlier this year, Trino added
+[new features](https://trino.io/blog/2022/05/05/tardigrade-launch.html) that
+simplifies running batch queries on Trino by introducing a fault-tolerance mode
+that only requires restarting a task on failure rather than a stage or the
+entire query.
+
+Batch queries don't typically involve humans in the loop and run asynchronously.
```
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-12-20-trino-summit-2022-astronomer-recap.md:21]: humans in the loop was a commonly used phrase in academic papers circa 2015 meaning it requires human intervention. I'm not sure how common it is now but yeah, probably better to just say that instead :)

File: _posts/2022-12-20-trino-summit-2022-astronomer-recap.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+These tasks may depend on each other and have a complex workflow. To orchestrate
+this complexity, this talk covers using Airflow's new Trino integration to
+run Trino batch queries to solve (almost) all your data problems.
+
+<!--more-->
+
+{% youtube xKDN7RUJ5i4 %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Astronomer.pdf">
+  Check out the slides here!
```
[the remark this answers is not in the document]
Comment (bitsondatadev) [reply _posts/2022-12-20-trino-summit-2022-astronomer-recap.md:31]: Done for both

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+title: "Using Trino with Apache Airflow for (almost) all your data problems"
+author: "Philippe Gagnon, Brian Olsen"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/astronomer.jpg
+---
+
+As we close in on the final talks from [Trino Summit 2022]({% post_url 
+2022-11-21-trino-summit-2022-recap %}), this next talk dives into how to set up
+Trino for batch processing. Trino has historically been well-known for providing
+fast adhoc analytics queries as opposed to long-running, resource intensive
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:12]: providing -> supporting? Maybe facilitating?

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+queries commonly classified as batch or ETL queries. This is due to the fact
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:13]: Try condensing with "resource intensive batch/ETL queries"

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+that Trino kills queries that run out of resources in favor of lower latency.
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:14]: "in favor" doesn't make much sense because the context doesn't answer the question "in favor of what"

Suggest "...run out of resources in order to prioritize faster query execution" or similar

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+Earlier this year, Trino added
+[new features](https://trino.io/blog/2022/05/05/tardigrade-launch.html) that
+simplify running batch queries using a new fault-tolerance execution mode. This
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:17]: Suggest rewording to "Trino added features to better support batch queries with a new fault-tolerant execution mode" and make "fault tolerant execution mode" the link text

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+mode backs up intermediate data during execution time and only requires
+restarting a task on failure rather than restarting a stage or the entire query.
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:19]: "data during execution time, allowing Trino to restart individual query tasks on failure rather than a query stage or the query itself."

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+
+Batch queries don't typically involve human intervention and run asynchronously.
+These tasks may depend on each other and have a complex workflow. To orchestrate
+this complexity, this talk covers using Airflow's new Trino integration to
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:23]: Suggest "This talk describes how to orchestrate this complexity using Airflo's new..."

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+
+{% youtube xKDN7RUJ5i4 %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Astronomer.pdf">
+  Check out the slides!
+</a>
+
+## Recap
+
+This talk covers a fictional trading scenario that initially follows a
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:36]: "In this talk, Philippe describes a fictional..."

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+traditional warehousing approach to storing data. This architecture has data
+sources that are queried and submitted as raw data into a centralized warehouse.
+Within the warehouse itself, the raw data is transformed into data ready to be
+consumed.
+
+This model enforces centralization which generally involves one team to run the
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:42]: "centralization, in which one team runs the platform and builds the integration"

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+platform and build the integration between producers and consumers. This team
+focuses on the aspects of the data platform which separates them further from
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:44]: "which further separates them"

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+the business use case. As source databases evolve, the central data team must
+keep up with these changes. As the teams that rely on the data infrastructure
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:46]: Maybe "data consumer teams"? To better differentiate from the bottleneck team you're describing

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+grow, this team commonly becomes a bottleneck.
+
+Trino allows you to move the queries as close as possible to the federated data
+sources. This removes the labor-intensive process of moving data into stages
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:50]: Condense with "to the federated data sources, removing the labor-intensive process"

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+before ingesting it into a central warehouse. This doesn't mean that data
+movement is no longer a necessity, but it does mean that the necessity shifts
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:52]: "but it does mean that the necessity" -> "but the necessity shifts"

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+closely with producers and stakeholders with a shared understanding of the
+domain. This increases data literacy and data availability throughout your
+organization.
+
+Trino is not only for fast adhoc analytics with a human in the loop, but now 
+provides a fault-tolerant execution mode that enables it to run resource
+intensive batch jobs. This, paired with the federation capabilities, make Trino
+able to ingest any data that can be represented in a tabular format. Users can
+implement user-defined functions and run transformations using SQL without
+involving intermediate systems.
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:65]: This now reads as an advertisement for FTE, rather than a talk recap. If possible, would suggest rewriting these paragraphs in particular to focus on the framing of the talk itself e.g. "In this scenario, we set up fault-tolerant execution to work with these federation capabilities"  and similar

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 20]
```diff
@@ -0,0 +1,93 @@
...
+
+To run Trino batch queries at scale requires building complex interdependencies
+between different tasks and often needs monitoring if there are any failures
+that occur. It also needs reactive automation to handle those instances. Apache
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:69]: "It also needs" -> "This configuration also demands"

Comment (jhlodin) [reply _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:69]: "those instances" -> "these instances of failure"

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 21]
```diff
@@ -0,0 +1,93 @@
...
+Airflow is an open-source platform for developing, scheduling, and monitoring
+batch-oriented workflows on systems like Trino.
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:71]: Missing a clause of how this relates to the previous sentence, like "...on systems like Trino, perfectly complementing the challenges of handling these intensive queries at scale."

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 22]
```diff
@@ -0,0 +1,93 @@
...
+
+The talk then covers:
+
+* Basic job creation
+* Sharded job creation that splits the large jobs into smaller tasks that
+  can easily be retried
+* Dynamic task mapping which splits jobs into smaller tasks based on data
+  observed at runtime
+* Data aware scheduling can schedule tasks based on interdependencies between
+  datasets
```
Comment (jhlodin) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:81]: It feels very off to have the majority of this post be an advertisement for FTE and Trino capabilities, then summarize all the Airflow-specific talk content in a bulleted list with no practical information. Needs a rewrite, even if it's just one or two paragraphs with the high level Airflow concepts.

**Review by jhlodin — CHANGES_REQUESTED** [review]
Some quick comments, but some larger revisions needed.

**Review by colebow — APPROVED** [review]
Gonna approve pending addressing Joe's feedback so you can merge once you've taken care of it. 👌

File: _posts/2022-12-21-trino-summit-2022-astronomer-recap.md [hunk 23]
```diff
@@ -0,0 +1,93 @@
...
+
+{% youtube xKDN7RUJ5i4 %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Astronomer.pdf">
+  Check out the slides!
+</a>
+
+## Recap
+
+In this talk, Philippe describes a fictional trading scenario that initially
```
Comment (colebow) [inline _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:36]: You need to introduce who Philippe is before referring to him by first name alone.

Comment (jhlodin) [reply _posts/2022-12-21-trino-summit-2022-astronomer-recap.md:36]: +1, the sentence above starting with "This talk describes" is a good place for that introduction
